File: LeetCode/Python/#20 Remove Element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        lens = nums.count(val)
        while lens != 0:
            nums.remove(val)
            lens -= 1

        return len(nums)


# Second
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        try:
            while True:
                nums.remove(val)
        except:
            return len(nums)

File: LeetCode/JavaScript/#20 Remove Element.js
/**
 * @param {number[]} nums
 * @param {number} val
 * @return {number}
 */
var removeElement = function(nums, val) {
    var len = nums.length;
    var res = 0;
    for(let i = len-1; i >=0; i--){
      if(nums[i]===val){
        nums.splice(i,1);
      }else{
        res++;
      }
    }
    return res;
};